rotkehlchen/chain/ethereum/modules/aave/common.py

```diff
@@ -133,43 +125,8 @@ def _get_reserve_address_decimals(asset: CryptoAsset) -> tuple[ChecksumEvmAddres
     return reserve_address, decimals
 
 
-class AaveHistory(NamedTuple):
-    """All events and total interest accrued for all Atoken of an address
-    """
-    events: list[AaveEvent]
-    total_earned_interest: dict[CryptoAsset, Balance]
-    total_lost: dict[CryptoAsset, Balance]
-    total_earned_liquidations: dict[CryptoAsset, Balance]
-
-
-class AaveInquirer():
-
-    def __init__(
-            self,
-            ethereum_inquirer: 'EthereumInquirer',
-            database: 'DBHandler',
-            premium: Optional[Premium],
-            msg_aggregator: MessagesAggregator,
-    ) -> None:
-        self.ethereum = ethereum_inquirer
-        self.database = database
-        self.msg_aggregator = msg_aggregator
-        self.premium = premium
-
-    def get_history_for_addresses(
-            self,
-            addresses: list[ChecksumEvmAddress],
-            to_block: int,
-            from_timestamp: Timestamp,
-            to_timestamp: Timestamp,
-            aave_balances: dict[ChecksumEvmAddress, AaveBalances],
-    ) -> dict[ChecksumEvmAddress, AaveHistory]:
-        """
-        Queries aave history for a list of addresses.
-
-        This function should be entered while holding the history_lock
-        semaphore
-        """
-        raise NotImplementedError(
-            'get_history_for_addresses() should only be implemented by subclasses',
-        )
+class AaveStats(NamedTuple):
+    """Total interest accrued for all Atoken of an address"""
+    total_earned_interest: dict[Asset, Balance]
+    total_lost: dict[Asset, Balance]
```

Comment:
Used `Asset` here instead of cryptoasset because the events have `Asset` and there is no need to resolve them